app/views/spree/admin/orders/_filters.html.haml

```diff
@@ -1,38 +1,40 @@
 %div{"data-hook" => "admin_orders_index_search"}
-  = search_form_for [:admin, @search], html: { name: "orders_form", "ng-submit" => "fetchResults()"} do |f|
```

Comment:
Is this `search_form_for` a Rails thing? I'm trying to understand what's the change that we did in this partial.

Reply:
`search_form_for` is an addition from the Ransack gem used for creating search forms. In order to use this alternate form helper here we have to construct a ransack search object in the controller and pass it to the view. So it's an additional dependency that adds more mess in the controller action, but actually it's no longer needed as we are passing the form data to Angular instead.

Reply:
So changing this to the basic `form_for` helped to tidy up the controller.